editor/src/components/canvas/multiplayer-presence.tsx

```diff
@@ -101,7 +101,7 @@ export const MultiplayerPresence = React.memo(() => {
 
   React.useEffect(() => {
     // when the room changes, reset
-    dispatch([switchEditorMode(EditorModes.selectMode(null, false, 'none'))])
+    setTimeout(() => dispatch([switchEditorMode(EditorModes.selectMode(null, false, 'none'))]), 0)
```

Comment:
wowza 😬

Reply:
oh yes, this is an unrelated fix to remove the flushSync errors